MediaBrowser.Controller/Entities/TV/Series.cs

```diff
@@ -344,6 +346,10 @@ public async Task RefreshAllMetadata(MetadataRefreshOptions refreshOptions, IPro
             }
 
             refreshOptions = new MetadataRefreshOptions(refreshOptions);
+            if (string.IsNullOrEmpty(refreshOptions.SearchResult.EpisodeGroupId))
+            {
+                this.EpisodeGroupId = null;
+            }
```

Comment:
Nitpicking, but ending curly braces should be followed by an empty line:
https://documentation.help/StyleCop/SA1513.html

Reply:
Done!